### ISSUE TITLE
More complex GAN models?

### ISSUE THREAD

**Issue by alelordelo** [JacopoMangiavacchi/MNIST-CoreML-Training#4]
Amazing!👊

Would it be possible to train more complex models like pix2pix?

**Comment by JacopoMangiavacchi** [comment]
Thank you Alexandre. I'm confident that specially with the new Apple Silicon more complex models could be trained with the SwiftCoreMLTools library.  That library already expose several layers, loss functions and optimizers available from the CoreML runtime.  If something specific is missing but available from the CoreML runtime it could be easily added to the library.

Please take a look at https://github.com/JacopoMangiavacchi/SwiftCoreMLTools and feel free eventually to contribute.
